generators/server/templates/pom.xml.ejs

```diff
@@ -363,6 +363,18 @@
             <artifactId>liquibase-mssql</artifactId>
         </dependency>
         <%_ } _%>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <%_ if (cucumberTests) { _%>
```

Comment:
Cucumber does not support JUnit 5 yet (see https://github.com/cucumber/cucumber-jvm/issues/1149).

JUnit 5 provides a way to use old ("vintage") APIs with junit-vintage, we will be able to remove this after JUnit 5 is added in Cucumber